foyer/tests/test_performance.py

```diff
@@ -23,7 +23,7 @@ def test_surface():
 
 
 @pytest.mark.skipif(not has_mbuild, reason="mbuild is not installed")
-@pytest.mark.timeout(45)
+@pytest.mark.timeout(60)
```

Comment:
Is lark slower?

Reply:
Yes, I think it's a little slower but it's also more powerful. This specific test was strange because it was consistently failing on only 1 of the 6 tests (macOS Python 2.7) so we decided just to bump it up the tolerance a little; 45s was already somewhat arbitrary and as long as the test runs in a workable amount of time it could be considered passing, IMHO.

Reply:
I switched to the LALR parser in b06d372f2737dd7af5a1046fee3b9636d036970e. Based on my tests this results in ~3x speed improvements (with the current test suite)